ios/Headless Universal Checkout/RNTPrimerHeadlessUniversalCheckoutRawDataManager.m

```diff
@@ -0,0 +1,21 @@
+//
+//  RNTPrimerHeadlessUniversalCheckoutRawDataManager.m
+//  primer-io-react-native
+//
+//  Created by Evangelos on 3/8/22.
+//
+
+#import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
+
+@interface RCT_EXTERN_MODULE(PrimerHeadlessUniversalCheckoutRawDatamanager, RCTEventEmitter)
```

Comment:
`*Manager`

Reply:
Added the suggestion so that it can be committed straightaway 👍 -> 🦅 👀 though 👌